keyboards/adpenrose/kintsugi/kintsugi.h

```diff
@@ -26,7 +26,7 @@
  * The second converts the arguments into a two-dimensional array which
  * represents the switch matrix.
  */
-#define LAYOUT_65_ansi_blocker( \
+#define LAYOUT_all( \
```

Comment:
Is there a reason for this name change to the layout macro? 

If not, then it should be reverted.

Reply:
The layout of this keyboard is not exactly a 65%. The OLED and the encoder take up the space os some keys, and I felt that this was more adequate.

Reply:
well, does it match the 65_ansi_blocker layout?  If so, then it should be left as is. 

Otherwise, `LAYOUT` should be used.

Reply:
No, it doesnt. Then I'll change everything to LAYOUT